cli/.angular-cli.json

```diff
@@ -0,0 +1,106 @@
+{
+    "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+    "project": {
+      "name": "ng-universal-demo"
+    },
+    "apps": [
+      {
+        "root": "src",
+        "outDir": "dist",
+        "assets": [
+          "assets",
+          "favicon.ico"
+        ],
+        "index": "index.html",
+        "main": "main.ts",
+        "polyfills": "polyfills.ts",
+        "test": "test.ts",
```

Comment:
Not sure if we should leave test config in, it can be removed but it would make it easier later on if someone were to add the full testing suit back in